modules/retail/pages/retail-install-setup.adoc

```diff
@@ -69,6 +69,12 @@ You will need to clone the repository locally to use it for building.
 Before you configure the branch server, ensure you have decided on networking topology, and know the minion ID of the branch server.
 For the information about the possible network topologies, see xref:retail:retail-network-arch.adoc[].
 
+[NOTE]
+====
+In case you plan to use the branch server as a monitoring server with Prometheus, be aware that Prometheus demands additional hardware ressources.
```

Comment:
```suggestion
In case you plan to use the branch server as a monitoring server with Prometheus, be aware that Prometheus demands additional hardware resources.
```